src/shared-components/dropdown/index.tsx

```diff
@@ -61,7 +61,7 @@ export const Dropdown = <T extends OptionType>({
   options,
   textAlign = 'left',
   value,
-  ariaLabel,
+  ...rest
```

Comment:
Just curious: Are we fine with any attribute? Or do we want to hold onto a list of accepted values? We could then update to validate `...rest` and reject any unknown attributes.